### PR TITLE
AC Test Fix - LRAC-10060 | master 

### PR DESCRIPTION
**Pull request by manuferreira**
https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1601670638&testrayRunId=1601674274 (Testray)
https://issues.liferay.com/browse/LRAC-10060 (Bug ticket)

### CODE REVIEW

**Comment by manuferreira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationWebBehavior.testcase [hunk 1]
```diff
@@ -737,12 +737,9 @@ definition {
 	}
 
 	@description = "Create a Web Behavior using 'has not' segment"
-	@ignore = "true"
 	@priority = "5"
-	test CanCreateWebBehaviorSegmentUsingHasNot {
-
-		// AC Bug Ticket: LRAC-10060
-
+	test CanCreateWebBehaviorSegmentUsingHasNot	{
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationWebBehavior.testcase:741]: Hey @manuferreira, I think the SF error is here

Comment (manuferreira) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsCreationWebBehavior.testcase]: I will check it out, thanks!!

**Review by ViniLopes20 — APPROVED** [review]

**Comment by manuferreira** [comment]
hey @alexcandidos I was thinking in opening another PR for removing these ones from quarantine, but now I think your approach would be better, it would save me time, thanks!

**Comment by manuferreira** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @manuferreira , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**